src/sidebars/favourites/geometry-service.js

```diff
@@ -77,12 +77,6 @@
     }
 
     class GeometryService {
-        windowsIntersect(openFinWindow, _window) {
-            var nativeWindow = openFinWindow.getNativeWindow();
-
-            return intersectHelper(getWindowPosition(nativeWindow), getWindowPosition(_window));
-        }
```

Comment:
They are, each only in one place (`elementIntersect`). However, I think it'll be good to keep them separate because of the separation of concerns.